test/base.py

```diff
@@ -47,11 +48,18 @@ def setup_class(cls):
 
     def config_template(self):
         bindir = TestPG.find_pgbin()
+
+        if hasattr(psycopg2.extras, "PhysicalReplicationConnection"):
```

Comment:
I'd much rather have explicit selection of the backup mode in individual test cases.